api/HTMLCanvasElement.json

```diff
@@ -191,7 +191,7 @@
             "description": "<code>bitmaprenderer</code> context",
             "support": {
               "chrome": {
-                "version_added": "56"
+                "version_added": "66"
```

Comment:
This should match https://developer.mozilla.org/en-US/docs/Web/API/ImageBitmapRenderingContext#browser_compatibility but doesn't, can you double check?

Reply:
It looks like this is a case of early interface exposure.  The tests for the interface are simple; `"ImageBitmapRenderingContext" in self`.  I've updated the tests in the collector and manually updated the data for the interface here.

Reply:
Aha. https://chromestatus.com/feature/5709799995998208 agrees with 66.